src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -32,12 +32,16 @@ export default function CSVFileImport({url, title}: CSVFileImportProps) {
 
   const uploadFile = async (e: any) => {
       // Get the presigned URL
+      const authorization_token = localStorage.getItem('authorization_token');
       const response = await axios({
         method: 'GET',
         url,
         params: {
           name: file.name,
           type: file.type
+        },
+        headers: {
+          Authorization: 'Basic ' + authorization_token
```

Comment:
In this case, If authorization_token is null, user will receive 403 status code. But expected code is 401.
If authorization_token is null, it is better not to pass "Authorization" header at all.